modules/ROOT/partials/integrations/blazor-tech-ref.adoc

```diff
@@ -188,19 +188,29 @@ In your component:
 
 === `LicenseKey`
 
-Specifies the {productname} license key. Required for self-hosted deployments of {productname}. This property is not required for deployments using the {cloudname}. For more information on licensing, see: xref:license-key.adoc[License key].
 
-*Type:* `+String+`
+include::partial$integrations/common/license-key-property.adoc[]
```

Comment:
Can't use the common license key partial in blazor because the variable is not the same. ie: LicenseKey vs licenseKey

Reply:
discussed with you, @tiny-ben-tran we agreed this looks good now.